app/models/user.rb

```diff
@@ -5,6 +5,8 @@ class User < ApplicationRecord
   validates :password, confirmation: { case_sensitive: true }
   validates_presence_of :name, :address, :city, :state, :zip
 
+  enum role: [:default_user, :merchant_employee, :admin_user]
+  
```

Comment:
Just note the enum can also be a hash with assigned values like  enum role: {default_user: 0, etc...}